src/sentry/utils/services.py

```diff
@@ -190,11 +202,11 @@ class InvalidBackend(Exception):
         function.
         """
 
-    class ServiceState(threading.local):
+    class State(threading.local):
         def __init__(self):
-            self.backends = {}
+            self.context = None
 
-    state = ServiceState()
+    __state = State()
```

Comment:
This is pseudo-private now since this is passed to the selector and callback functions explicitly.